lms/templates/courseware/course_navigation.html

```diff
@@ -65,6 +66,13 @@
                 </p>
             </div>
             % endif
+            % if user.is_staff and not CourseEnrollment.is_enrolled(user, course.id):
+              <button class="staff-register-button" data-endpoint="${enrollment_url}" data-course-id="${unicode(course.id)}">
+                ${_("Register for course")}
+              </button>
+              <span class="button-message">(This button is only visible to global staff users)</span>
```

Comment:
Should be internationalized (not that I'm psyched about this feature... why is it being introduced?).